### ISSUE TITLE
seismic reverse-shooting [rt.marine-geo.org #2778]

### ISSUE THREAD

**Issue by nevilleshane** [iedadata/geomapapp#250]
Migrated from [rt.marine-geo.org#2778](http://rt.marine-geo.org/Ticket/Display.html?id=2778) (status was 'resolved')

Requestors:
* ra@ldeo.columbia.edu

From ra@ldeo.columbia.edu on 2011-08-12 19:44:59
:
```

GMA does not allow input of reverse-shot seismic lines, where the
shotpoint values decrease.  When the nav file is sorted to be sp increasing,
the line displays but the cursor in the main map window will not
follow the cursor movements in the lower panel.
rose anne

In other words:
The only problems I saw when I loaded reverse lines (example w1379ar)
is that the map tracking is screwed up.  So when you mouse-over the seismic,
the crosshairs on the map get confused.  Everything else went fine 
including jpegs.

Ray




```

From nshane@ldeo.columbia.edu on 2020-02-24 15:21:53
:
```
Ticket migrated to github as https://github.com/iedadata/geomapapp/issues/23
```
